src/highlighting.c

```diff
@@ -1685,6 +1687,10 @@ gboolean highlighting_is_string_style(gint lexer, gint style)
 				style == SCE_NIM_TRIPLE ||
 				style == SCE_NIM_TRIPLEDOUBLE ||
 				style == SCE_NIM_STRINGEOL);
+
+		case SCLEX_ZIG:
+			return (style == SCE_ZIG_STRING ||
+				style == SCE_ZIG_MULTISTRING);
```

Comment:
shouldn't this include `CHARACTER` and `ESCAPECHAR`?  I didn't check the lexer yet

Reply:
Correct, done.